Bottom: 29f652f1245401a0a9d8cf0755d0b8525993bd2e
Top:    06e58fa3ef0c4bd7641eb90e820b82f024c9d5d0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-01 01:44:27 +0100

Refresh of polygon1-c-temporarily-disable

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 5a91370..84b30ea 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -600,7 +600,8 @@ adjust_tree (rtree_t * tree, struct seg *s)
   return 0;
 }
 
-#if 0
+
+#if 0  /* TURNS OUT DISABLING THIS ACTUALLY (VERY MARGINALLY) SPEEDS THINGS UP! */
 /*
  * seg_in_region()
  * (C) 2006, harry eaton
@@ -846,7 +847,7 @@ contour_bounds_touch (const BoxType * b, void *cl)
       info.tree = rtree_over->tree;
       if (info.tree)
 	if (UNLIKELY (r_search (info.tree, &info.s->box,
-				NULL/* seg_in_region */, seg_in_seg, &info)))
+				NULL /* seg_in_region */, seg_in_seg, &info)))
 	  assert (0); /* XXX: Memory allocation failure */
     }
   while ((av = av->next) != &looping_over->head);
